docs/config.md

```diff
@@ -42,20 +36,24 @@ batch_sizes:
     step: 2
 ```
 
-This YAML represents the array `[2, 4, 6]`.
-* Other types are described in the end of this file.
+The above YAML declares the value of batch_sizes to be an array `[2, 4, 6]`.
+
+## CLI and YAML Config Options
+
+A list of all the configuration options supported by both the CLI and YAML config file are shown below.
```

Comment:
Can you add this before the scheme?

"The scheme for the configuration file is described below. Brackets indicate
that a parameter is optional. For non-list and non-object parameters the
value is set to the specified default."

Reply:
Can you explain how they can convert the field names to CLI flags?

"You need to convert the snake case to kebab case for the config field to become the CLI flag."

Reply:
Ah yes I think I meant to cut and paste that first paragraph but I forgot. Good catch thanks.